pennylane/beta/tapes/qnode.py

```diff
@@ -299,16 +299,10 @@ def _get_parameter_shift_method(device, interface):
         model = device.capabilities().get("model", None)
 
         if model == "qubit":
-            return QubitParamShiftTape, interface, "analytic"
+            return QubitParamShiftTape, interface, "best"

```

Comment:
This was a bug in the previous PR (setting it as 'analytic' fails to allow it to fallback to finite-differences if needed).